lib/srv/regular/sshserver.go

```diff
@@ -2056,6 +2072,11 @@ func (s *Server) parseSubsystemRequest(req *ssh.Request, ctx *srv.ServerContext)
 	case s.proxyMode && strings.HasPrefix(r.Name, "proxysites"):
 		return parseProxySitesSubsys(r.Name, s)
 	case r.Name == sftpSubsystem:
+		if err := ctx.CheckFileCopyingAllowed(); err != nil {
+			writeStderr(ch, err.Error()+"\n")
```

Comment:
Do we really need the `\n`? I don't see any other usage in our code where we would do that.

Reply:
Without the newline the error message from Teleport would bleed into the OpenSSH error message without a clear delineation between the two

Reply:
Can add a comment with the explanation?